device/core/src/internal_client.ts

```diff
@@ -337,6 +339,68 @@ export abstract class InternalClient extends EventEmitter {
     }, abandonCallback);
   }
 
+  /**
+   * Sends a property update patch to the Azure IoT Hub or Azure IoT Edge Hub service.
+   *
+   * @param {ClientPropertyCollection} propertyCollection A ClientPropertyCollection object representing the property patch.
+   * @param {(err: Error) => void}     done               A callback used to get notified of the success or failure of the operation.
+   *                                                      If no callback is specified, a promise is returned instead.
+   */
+  updateClientProperties(propertyCollection: ClientPropertyCollection): Promise<void>;
+  updateClientProperties(propertyCollection: ClientPropertyCollection, done: (err: Error) => void): void;
+  updateClientProperties(propertyCollection: ClientPropertyCollection, done?: (err: Error) => void): Promise<void> | void {
+    return callbackToPromise((_callback) => {
+      this.getTwin((err, twin) => {
+        if (err) {
+          _callback(err);
+          return;
+        }
+        (twin as any)._updateReportedProperties(propertyCollection.backingObject, _callback);
+      });
+    }, done);
+  }
+
+  /**
+   * Registers a listener to get invoked with a writable property patch whenever the device receives a writable property request.
+   *
+   * @param {(properties: ClientPropertyCollection) => void} callback The callback to get invoked on a writable property request.
```

Comment:
fix this js doc to include a - between callback and The callback. Like: 

@param {string} title - The title of the book.

Reply:
Fixed here and in a bunch of other places